qase-playwright/changelog.md

```diff
@@ -1,3 +1,17 @@
+# playwright-qase-reporter@2.0.2
+
+## What's new
+
+Added new annotation for test case ignoring `qase.ignore()`.
+This annotation will mark the test as ignored in the Qase.io test run.
```

Comment:
it's important to note that test will not be ignored by Playwright itself, just won't be reported 
```suggestion
Added new annotation `qase.ignore()`.
Tests marked with it will run as usual but won't appear in the Qase report.
```

Reply:
fixed